packages/element-templates-json-schema-shared/src/defs/base.json

```diff
@@ -6,25 +6,30 @@
     "properties"
   ],
   "properties": {
+    "$schema": {
+      "$id": "#/$schema",
+      "type": "string",
+      "description": "Reference to the Element Template JSON schema this document follows."
+    },
     "name": {
       "$id": "#/name",
       "type": "string",
-      "description": "The name of the element template."
+      "description": "Template name. Shown during template selection and editing of a templated element . Required."
     },
     "id": {
       "$id": "#/id",
       "type": "string",
-      "description": "The identifier of the element template."
+      "description": "Template identifier. Required."
     },
     "description": {
       "$id": "#/description",
       "type": "string",
-      "description": "The description of the element template."
+      "description": "Template description. Shown during template selection and editing of a templated element."
     },
     "version": {
       "$id": "#/version",
       "type": "integer",
-      "description": "Optional version of the template. If you add a version to a template it will be considered unique based on its ID and version. Two templates can have the same ID if their version is different."
+      "description": "Optional version of the template. If you add a version to a template, it will be considered unique based on its ID and version. Two templates can have the same ID if their version is different."
```

Comment:
```suggestion
      "description": "Optional version of the template. If you add a version to a template, it will be considered unique based on its ID and version. Two templates can have the same ID if their versions are different."
```